src/Libraries/OrchardCore.Commerce.Abstractions/ViewModels/ShoppingCartLineViewModel.cs

```diff
@@ -17,7 +17,9 @@ public class ShoppingCartLineViewModel : ILineItem
     public Amount UnitPrice { get; set; }
     public Amount LinePrice { get; set; }
 
-    public IDictionary<string, JToken> AdditionalData { get; } = new Dictionary<string, JToken>();
+#pragma warning disable CA2227
+    public IDictionary<string, JToken> AdditionalData { get; set; } = new Dictionary<string, JToken>();
+#pragma warning restore CA2227
```

Comment:
Can you use `init` instead of `set` like this?
```suggestion
    public IDictionary<string, JToken> AdditionalData { get; init; } = new Dictionary<string, JToken>();
```